cli/src/main/java/org/commonjava/maven/ext/cli/Cli.java

```diff
@@ -87,7 +87,7 @@
 public class Cli
 {
     private static final File DEFAULT_GLOBAL_SETTINGS_FILE =
-        new File( System.getenv( "M2_HOME" ), "conf" + File.separator + "settings.xml" );
+        new File( System.getProperty( "maven.home" ), "conf" + File.separator + "settings.xml" );
```

Comment:
https://maven.apache.org/guides/introduction/introduction-to-profiles.html

Global
- Defined in the global Maven-settings (${maven.home}/conf/settings.xml).

Reply:
Looking at https://maven.apache.org/shared/maven-invoker/usage.html while in general I think this should be ok I am beginning to think especially if we want to test on older versions occasionally we might need some backwards compatibility here. Perhaps belt/braces approach and set all of them?!!

Reply:
Similar to how https://maven.apache.org/shared/maven-invoker/usage.html does ?

Reply:
Lets discuss today on IRC